pkg/scheduler/framework/plugins/imagelocality/image_locality.go

```diff
@@ -107,6 +107,13 @@ func sumImageScores(nodeInfo *framework.NodeInfo, pod *v1.Pod, totalNumNodes int
 			sum += scaledImageScore(state, totalNumNodes)
 		}
 	}
+	for _, volume := range pod.Spec.Volumes {
```

Comment:
Not sure if should check the "ImageVolume" feature gate at here, and only account volume when it enabled.

Reply:
why?

Reply:
None from me. Let's wait for other reviewers

Reply:
Well, actually I'm not sure about this.
We may have a scheduler without this fg, but both api-server and kubelet has it enabled. 
In this case, kubelet will pull the images anyway.
I prefer to calculate them anyway. /cc @macsko  What's your opinion?